filestore/handlers.py

```diff
@@ -60,15 +60,15 @@ def __init__(self, fpath, template, filename, frame_per_point=1):
         self._image_sequence = pims.ImageSequence(self._filename)
 
     def __call__(self, point_number):
-        start, stop = point_number * self._fpp, (point_number + 1) * self._fpp
+        start= int(point_number * self._fpp)
+        stop = int((point_number + 1) * self._fpp)
         if stop > len(self._image_sequence):
             # if asking for an image past the end, make sure we have an up to
             # date list of the existing files
             self._image_sequence = pims.ImageSequence(self._filename)
-        if stop > len(self._image_sequence):
-            # if we _still_ don't have enough files, raise
-            raise IntegrityError("Seeking Frame {0} out of {1} frames.".format(
-                stop, len(self._image_sequence)))
+            if stop > len(self._image_sequence):
+                # if we _still_ don't have enough files, raise
+                raise IntegrityError("Seeking Frame {0} out of {1} frames.".format(stop, len(self._image_sequence)))
```

Comment:
On Tue, 17 Nov 2015, Dan Allan wrote:
> In filestore/handlers.py:
...
> > +                raise IntegrityError("Seeking Frame {0} out of {1} frames.".format(stop, len
> (self._image_sequence)))
> 
> I didn't mean to change this, actually, but it does make more sense this way. Google "PEP8" and
> read up on that. Then add a new line after format( and indent the rest four spaces in relative
> to the line above it.


Align with the '(', or when the line is too long, indent < > 4 (commonly
8, but whatever is most readable and distinguishes continuation lines
from control flow indentation).  I'd do this:

                 raise IntegrityError(
                         "Seeking Frame {0} out of {1} frames.".format(
                                 stop, len(self._image_sequence)))


https://www.python.org/dev/peps/pep-0008/#indentation
Although the 'pep8' style check command seems to want continuation lines indented 4? :(